pkg/helper/pipelinerun/description/description.go

```diff
@@ -62,6 +62,12 @@ STARTED	DURATION	STATUS
 {{- end }}
 {{- end }}
 
+{{decorate "timeout" ""}} {{decorate "underline bold" "Timeout\n"}}
```

Comment:
My thought would be to move this up to where, for instance, ServiceAccount, is rendered. The lower portion of the template is usually for lists, such as TaskRuns in this case.

Reply:
I have moved timeout under Service Account